deployment/README.md

```diff
@@ -100,9 +102,9 @@ See the login details and browser addresses above.
 
 ### Deployment
 
-Afterwards you can [deploy](/deployment/deployment.md) it on your server:
+Afterwards you can [deploy](./deployment.md) it on your server:
 
-- [Kubernetes with Helm](/deployment/kubernetes/README.md)
+- [Kubernetes with Helm](./src/kubernetes/README.md)
```

Comment:
Why relative links here if all others are absolut?

Reply:
In case linked markdown files are in the same directory, relative paths work for both Github and Vuepress.